README.md

```diff
@@ -160,6 +160,11 @@ git clone https://github.com/PABannier/bark.cpp.git
 cd bark.cpp
 ```
 
+### Pull submodules (ggml)
+```bash
+git submodule update --init --recursive
```

Comment:
Not sure this is the right command, since `bark.cpp` currently depends on a specific commit hash of `ggml`. By updating the submodule, we are breaking compatibility.

I think this is the right approach to proceed as it does not update the submodule:

```suggestion
git clone --recursive https://github.com/PABannier/bark.cpp.git
```